src/main/java/io/github/externschool/planner/service/TeacherServiceImpl.java

```diff
@@ -87,4 +89,28 @@ public void deleteTeacherById(Long id) {
             teacherRepository.deleteById(id);
         }
     }
+
+    @Override
+    @Scheduled(cron = "0 0 01 ? * SAT")
+    public void updateTeacherSchedule(){
+
```

Comment:
in fact you can put as many empty spaces between lines as you wish, but I don't like such a sparse representation - when your method is really big you have to use scrolling to get it all